pkg/assets/handlers.go

```diff
@@ -104,7 +108,7 @@ func HTML5ModeHandler(h http.Handler) http.Handler {
 	})
 }
 
-func GeneratedConfigHandler(osAddr string, osPrefix string, k8sAddr string, k8sPrefix string, h http.Handler) http.Handler {
+func GeneratedConfigHandler(osAddr, osPrefix, k8sAddr, k8sPrefix, oauthAuthURL, oauthClientID string, h http.Handler) http.Handler {
```

Comment:
Got a few arguments, eh?  Is this a one off or do these get passed around elsewhere?